nucypher/cli/painting.py

```diff
@@ -56,7 +56,7 @@ def build_fleet_state_status(ursula) -> str:
         fleet_state = '{checksum} ⇀{nickname}↽ {icon}'.format(icon=fleet_state_icon,
                                                               nickname=fleet_state_nickname,
                                                               checksum=fleet_state_checksum)
-    elif ursula.known_nodes.checksum is not NO_KNOWN_NODES:
+    elif ursula.known_nodes.checksum is NO_KNOWN_NODES:
```

Comment:
I don't get this pattern I've seen multiple times in the codebase:
```python
if foo is bar:
    do_something()
elif foo is not bar:
    do_another_thing()
```
Sometimes with `is`, but sometimes with `==`; it doesn't matter. The thing is that the `elif` could be safely substituted by an `else`, since you're testing the logical opposite. Is this pattern a Python thing?

In this case, there's even an `else` clause after the `elif`, which doesn't make any sense: it's unreachable code.

Reply:
It may be because it is possible for the value to be `None` or some equivalent - I assumed that was why there was an `elif` and an `else`. If not, then the `elif` can be replaced with just `else`

Reply:
If `ursula.known_nodes.checksum` is `None`, then `ursula.known_nodes.checksum is foo` will only be true when `foo` is None, too. And conversely, if `ursula.known_nodes.checksum` is `None`, `ursula.known_nodes.checksum is not bar` will always be true, unless `bar` is `None`.

Reply:
Ah! - good point - yeah there is unreachable code.

Reply:
Haha, yeah I have no idea.  This has the clear signature of @KPrasch having written some code about an hour after he started to really need some sleep.  What's happening here, man?  Why are you even making a string in the case that there are no known nodes??  Can't you just return the constant or something?